api/src/oscc.c

```diff
@@ -418,23 +496,38 @@ void oscc_update_status( )
             }
             else
             {
-                if ( obd_frame_callback != NULL )
+                if ( obd_frame_callback != NULL && vehicle_can_socket < 0 )
                 {
                     obd_frame_callback( &rx_frame );
                 }
             }
 
-            ret = read( can_socket, &rx_frame, CAN_MTU );
+            ret = read( oscc_can_socket, &rx_frame, CAN_MTU );
         }
     }
+
+    if ( vehicle_can_socket != -1 )
+    {
+      int veh_ret = read( vehicle_can_socket, &rx_frame, CAN_MTU );
```

Comment:
`veh_ret` is bytes read from the vehicle CAN bus, correct?

Reply:
Yes, maybe `ret` and `veh_ret` should be renamed to `oscc_bytes` and `vehicle_bytes`...